src/main/java/heronarts/lx/midi/surface/MidiFighterTwister.java

```diff
@@ -337,12 +350,57 @@ public void onParameterChanged(LXParameter parameter) {
       }
     }
 
+    private boolean isKnobRelative(int index) {
+      // Fixed for now but could be expanded to poll MFT encoder settings on startup with sysex
+      return true;
+    }
+
     private void onKnob(int index, double normalized) {
       if (this.knobs[index] != null) {
         this.knobs[index].setNormalized(normalized);
       }
     }
 
+    private final static double KNOB_INCREMENT_AMOUNT = 1.0/128.0;
+
+    private void onKnobDecrement(int index) {
+      LXListenableNormalizedParameter knob = this.knobs[index];
+      if (knob != null) {
+        if (knob instanceof DiscreteParameter) {
+          // Move after a set number of ticks in the same direction
+          this.knobTicks[index] = LXUtils.min(this.knobTicks[index], 0) - 1;
+          if (this.knobTicks[index] == 0-KNOB_TICKS_PER_DISCRETE_INCREMENT) {
+            this.knobTicks[index] = 0;
+            ((DiscreteParameter)knob).decrement();
+          }
+        } else {
+          onKnobIncrement(knob, 0-KNOB_INCREMENT_AMOUNT);
+        }
+      }
+    }
+
+    private void onKnobIncrement(int index) {
```

Comment:
Feels really repetetive with the other mode. I'm not strictly opposed but a single method with a boolean param for inc vs. dec seems like it'd save code here.

Reply:
Good call.  Will change.